spec/emerald/parser_spec.rb

```diff
@@ -100,4 +100,30 @@
                           Emerald::Number.new(30)])
     end
   end
+
+  describe "string parsing" do
```

Comment:
What about parsing these strings?

```ruby
"this is a quote -> \" <- "
```

```ruby
"this is a slash then a quote -> \\\" <- "
```

You can do this with regex, though it's quite hard to wrap your head around. :)